TabnineSelfHosted/src/main/java/com/tabnineSelfHosted/loginNotifier.kt

```diff
@@ -0,0 +1,22 @@
+package com.tabnineSelfHosted
+
+import com.intellij.notification.NotificationType
+import com.intellij.openapi.components.ServiceManager
+import com.intellij.openapi.diagnostic.Logger
+import com.tabnineCommon.general.StaticConfig.TABNINE_TIMED_NOTIFICATION_GROUP
+import com.tabnineSelfHosted.binary.lifecycle.UserInfoService
+
+fun showUserLoggedInNotification() {
+    val userInfo = ServiceManager.getService(UserInfoService::class.java).fetchAndGet()
+    // show notification only if the user is part of a team
+    if (userInfo?.team == null) {
+        Logger.getInstance(showUserLoggedInNotification().javaClass)
+            .debug("Unable to get user info or no team. Info: ${userInfo ?: "null"}")
```

Comment:
why null? why not userInfo?

Reply:
User info might be null when still no answer from the binary